src/03_modules.py

```diff
@@ -10,22 +10,31 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+['']
```

Comment:
I know you know this stuff. But they want to see that you can use the print statement. 
import sys
for arg in sys.argv: 
       print(arg)